nburr-houston.md

```diff
@@ -0,0 +1,6 @@
+#Houston, TX
+
+#Places to eat
+1. Mellow Mushroom for good pizza
+1. The Boot for good crawfish
+1. Captain Tom's has awesome gumbo
```

Comment:
For great oysters, try Captain Tom's.  They'll put an oyster in your beer (Michelada) and ring a bell if you drink it.